app/build.gradle

```diff
@@ -82,6 +83,11 @@ task jacocoUnifiedTestReport(type: JacocoReport, dependsOn: ['connectedAndroidTe
 
 dependencies {
 
+    // This is for desugaring modern java.time API for older Android API versions (Do not remove!)
+    coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:1.2.2")
```

Comment:
What is this used for?

Reply:
So I am using LocalDate for comparing Dates and LocalDate can only be used by newer Android versions. Because of this, whenever GitHub Actions is running, the linter will fail every time due to LocalDate not working on older Android versions. 

So to fix this issue, I decided to introduce a desugaring dependency that converts LocalDate for older version so the linter doesn't burst into a dozen flames every time a commit is made to the Android repo. 

I know, it's dumb. And I am fully open to better ideas.